docs/topics/which_historical_backend.md

```diff
@@ -0,0 +1,104 @@
+
+Regarding storage capabilities, the data can be uploaded to the FileStore, a simple file manager, or to the DataStore, which is based on [PostgreSQL](http://www.postgresql.org).
+
+Apart from the open data capabilities of CKAN, it must be highlighted the visualization tools it provides in the form of viewers (other [viewers](http://docs.ckan.org/en/latest/maintaining/data-viewer.html) could be added upon request):
+
+* **Text View** and **Image View**. It simply displays the content of text and image files, respectively, within the FileStore. Special formats such as XML or Json have syntax highlighted.
+* **Recline View**. This is the visualization extension used for context datasets created by the platform in the DataStore. The data can be displayed as a grid, as a 2-axis graph or as a geolocation map (if the data contains some geolocation data).
+
+CKAN does not provide any analytical tool
+
+And equals than HDFS, CKAN allows for uploading you own datasets (open or not).
+
+[Top](#top)
+
+##STH Comet
+As its name denotes, this MongoDB-based backend is used for short-term historical persistence. By short-term must be understood months, days or even days.
+
+It exposes a REST API for data querying. FIWARE service and FIWARE service path must be given in the form of Http headers, together with a authentication token. This means this is a tol for private consumption of data.
+
+An interesting detail regarding STH Comet API that may lead you for choosing it is not only raw records can be retrieved; but aggregated ones as well, which can be considered analytics into some extent:
+
+* Sum of a subset of samples.
+* Square sum of a subset of samples.
+* Maximum sample in a subset of samples.
+* Minimum sample in a subset of samples.
+* Number of samples 
+
+Subsets are parameterized by certain range of time and a resolution.
+
+No visualization tools are given with STH Comet. Nevertheless, its REST API could be integrated by a skilled developer with some tools such as [Grafana](http://grafana.org/).
+
+No other data different than the one stored by the platform in MongoDB can be queried.
+
+[Top](#top)
+
+##Summary
+|Backend|Capacity|Privacy|Analytics|Visualization|Static upload|
+|---|---|---|---|---|---|
+|HDFS|Scalable (1)|Per Unix user and group (2)|Apache MapReduce (3)<br>Apache Hive (3)<br>Apache Pig (3)|Apache Hue's FileBrowser (3)|Yes|
+|MicroStrategy|?|Per database (4)|Many algorithms|Many dashboards|No|
+|CKAN|?|Per dataset (5)|-|CKAN text view<br>CKAN image view<br>CKAN recline view|Yes|
+|STH Comet|?|Per FIWARE service and FIWARE service path|Raw and aggregated querying API|-|No|
```

Comment:
Maybe the posibility to integrate visualization tools (as Graphana) could be mentioned instead of "-".